pkg/discovery/worker/compliance/compliance_processor.go

```diff
@@ -0,0 +1,152 @@
+// Package compliance parses different compliance rules in Kubernetes, creates commodityDTOs and
+// then updates corresponding entityDTOs.
+package compliance
+
+import (
+	"errors"
+	"fmt"
+
+	sdkbuilder "github.com/turbonomic/turbo-go-sdk/pkg/builder"
+	"github.com/turbonomic/turbo-go-sdk/pkg/proto"
+)
+
+// ComplianceProcessor provides methods to quickly find an entityDTO and updating its fields, such as commodities bought
+// and commodities sold based on different compliance rules.
+type ComplianceProcessor struct {
```

Comment:
ComplianceProcessor is part of [AffinityProcessor](https://github.com/turbonomic/kubeturbo/pull/76/files/64ad6997521ec0945fe0093269b7e39e71c78d46#diff-2e3c67eca8d78866a8768e73f676c73cR28)